src/applications/vaos/tests/e2e/fixtures/MockFacility.js

```diff
@@ -0,0 +1,17 @@
+export class MockFacility {
```

Comment:
There's a new Mocks Facility created in `src/applications/vaos/tests/mocks/unit-test-helpers.js`. Maybe we could create a universal helper file that are reusable for both e2e and unit tests?